src/Components/Navbar.js

```diff
@@ -1,14 +1,49 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { NavLink } from 'react-router-dom';
 import { toggleLogin } from '../Pages/LoginPage';
+import { toggleSignup } from '../Pages/SignupPage';
+import { logOutUser } from '../Redux/user/UserReducer';
+// import { toggleSignup } from '../Pages/SignupPage';
```

Comment:
- Commented is better removed than left inline.

Reply:
@hsztan Done. good catches.